locales/en-US/messages.properties

```diff
@@ -15,10 +15,13 @@ newProjectBtn=Start a project from scratch
 contactEmail=Email Us
 contactTwitter=Talk to us on Twitter
 readTheDocs=Read the Docs
-contribute=Contribute on Github
+contribute=Contribute on GitHub
```

Comment:
I think we were told that changes to existing strings required new keys vs. just new values, right?  If so, all these updates are wrong.  You also need to version the keys, e.g., `contribute-2` or something.